CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Bearded.Utilities
+
+You want to help us improve our little library? ~~That's awesome!~~ You're awesome!
+
+Whether you have a feature request, found a bug, want to add some code of your own or even just have a question - all of these are very welcome.
+
+Our library is pretty small, so let's try and go easy on the rules. :)
+
+
+## The basics
+
+- **Have a question?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+- **Found a bug?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+- **Would you like to request a feature, change or improvement?** [Open an issue!](https://github.com/beardgame/utilities/issues/new/choose)
+
+You could also always check out list of issues and see if there's any discussions you want to share your thoughts on.
+
+
+## You'd actually like to make changes yourself?
+
+Wow, that's even better!
+
+In principle this as easy as forking the repository, making some changes and [opening a pull request](https://github.com/beardgame/utilities/compare). From there, we'll review your PR as soon as we can manage, ask questions or request changes if necessary, and once we're happy merge it into our main branch. It's that easy!
```

Comment:
Who is “we”? In particular, will they have to assign a reviewer? And if so, how to pick one?

Reply:
I think it's reasonably clear that 'we' means 'maintainers'/'the people working on/owning this library'. I don't see why people have to pick one or what it would add if they did. We can figure out between ourselves how to handle that when/if it happens. If it ever becomes a problem, we can fix it then.